### ISSUE TITLE
survey-knockout-ui v1.9.18 doesn't build with TypeScript

### ISSUE THREAD

**Issue by dsimon-onventis** [surveyjs/survey-library#4087]
### Are you requesting a feature, reporting a bug or asking a question?

bug

### What is the current behavior?

Package `survey-knockout-ui` v1.9.18 doesn't build with TypeScript, with the following error:

```
node_modules/survey-knockout-ui/survey-knockout-ui.d.ts(16,25): error TS2300: Duplicate identifier 'Model'.
node_modules/survey-knockout-ui/survey-knockout-ui.d.ts(18,10): error TS2300: Duplicate identifier 'Model'.
```

### What is the expected behavior?

Package `survey-knockout-ui` builds properly with TypeScript (like it did with v1.9.14)

### Specify your

* browser: Chrome
* browser version: 98
* surveyjs platform (angular or react or jquery or knockout or vue): knockout
* surveyjs version: 1.9.18

**Comment by andrewtelnov** [comment]
@dsimon-onventis I will take a look.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@dsimon-onventis I have fixed it by the commit above. The bug was in our d.ts generator.
We will include the fix in v1.9.19.

Thank you,
Andrew
